### ISSUE TITLE
Feature request: Multiply blend mode

### ISSUE THREAD

**Issue by foguetes** [airbnb/lottie-web#1875]
I did an experimentation with blend modes in Lottie

It seems multiplying works fine when there are elements in the background of the lottie animation
https://lottiefiles.com/share/4RX8Gt

but when I hide those elements and place a background image via html it doesn't work
https://codepen.io/foguetes/pen/VwwRLBv

I would be great if the animation could interact with the background without any modification

**Comment by bodymovin** [comment]
I think that is not feasible beyond the lottie library.
Have you found any examples where svg content can be blended with backgrounds?

**Comment by foguetes** [comment]
Yes.
https://codepen.io/foguetes/pen/eYYXMap

**Comment by bodymovin** [comment]
I see what is happening. If you remove the transform property of the svg it will work. 
You can do so in multiple ways. One is by passing a property `className` to the render options with a class and overriding the transform value.

**Comment by foguetes** [comment]
ok. could you still improve the documentation though?
Lottie is one of the greatest tools with one of the worst Wikis.
My guess is if the documentation was thorough you would spend a lot less time replying to issues on Github.

**Comment by bodymovin** [comment]
I agree that documentation needs a lot of improvement. But how and where do you think this information should be added?

**Comment by foguetes** [comment]
If I visit the official Lottie airbnb page I get a 404 which doesn't really look good on them. I'm not a big fan of the Github wiki as well. A Lottie MediaWiki would be ideal, I believe, since it's easy to browse categories and hyperlinking articles.

**Comment by kudanai** [comment]
Hello,

looks like this issue has become stale now.
Since then there have been a few external and community driven documentation projects. For example:
* [https://github.com/lottie-animation-community/docs](https://github.com/lottie-animation-community/docs)
* [https://lottiefiles.github.io/lottie-docs/](https://lottiefiles.github.io/lottie-docs/)

Hopefully these address the issues somewhat better
